src/Streamly/Internal/Data/Stream/Ahead.hs

```diff
@@ -49,7 +49,7 @@ import qualified Data.Heap as H
 
 import Streamly.Internal.Control.Concurrent
     (MonadRunInIO, MonadAsync, askRunInIO, restoreM)
-import Streamly.Internal.Data.Stream.StreamK.Type (Stream)
+import Streamly.Internal.Data.Stream.StreamK.Type (StreamK)
```

Comment:
We do not need to change the deprecated modules, they can continue using Stream, we should only make minimal changes to deprecated code.